code/ui/alivehud/inspectmenu/InspectMenu.cs

```diff
@@ -39,7 +40,7 @@ public InspectMenu()
             _confirmationPanel = new ConfirmationPanel(this);
         }
 
-        public void InspectCorpse(TTTPlayer deadPlayer, bool isIdentified)
+        public void InspectCorpse(TTTPlayer deadPlayer, bool isIdentified, ConfirmationData confirmationData, string killerWeapon = null, string[] perks = null)
```

Comment:
Is there a reason why we can't package `KillerWeapon` and `Perks` inside of `confirmationData`? As well as `isIdentified`?

Reply:
https://github.com/TTTReborn/tttreborn/pull/134#discussion_r671869036

Reply:
So we could pass in `bool isIdentified`